lib/search/ui/widgets/search_bar.dart

```diff
@@ -0,0 +1,190 @@
+}
+
+class _SearchBarViewState extends State<SearchBarView> {
+  final TextEditingController _searchController = TextEditingController();
+  Timer? _debounce;
+
+  @override
+  void dispose() {
+    _debounce?.cancel();
+    _searchController.dispose();
+    super.dispose();
+  }
+
+  void _onSearchChanged(String query) {
+    if (_debounce?.isActive ?? false) _debounce?.cancel();
+    _debounce = Timer(const Duration(milliseconds: 300), () {
+      if (query.isNotEmpty) {
+        context.read<SearchCubit>().search(query);
+      } else {
+        context.read<SearchCubit>().clearSearch();
+      }
+    });
+  }
+
+  void _clearSearch() {
+    _searchController.clear();
+    FocusScope.of(context).unfocus();
+    context.read<SearchCubit>().clearSearch();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Container(
+          padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 12),
+          decoration: BoxDecoration(
+            color: Theme.of(context).cardColor,
+            borderRadius: BorderRadius.circular(35),
+            border: Border.all(color: ThemeColors.blueDroidconColor),
+          ),
+          child: Row(
+            children: [
+              const Icon(Icons.search,
+                  color: ThemeColors.blueGreenDroidconColor),
+              const SizedBox(width: 8),
+              Expanded(
+                child: TextField(
+                  controller: _searchController,
+                  decoration: InputDecoration(
+                    hintText: 'Search sessions or speakers',
+                    hintStyle: TextStyle(color: Theme.of(context).hintColor),
+                    border: InputBorder.none,
+                  ),
+                  onChanged: _onSearchChanged,
+                ),
+              ),
+              if (_searchController.text.isNotEmpty)
+                IconButton(
+                  icon: const Icon(Icons.cancel,
+                      color: ThemeColors.orangeDroidconColor),
+                  onPressed: _clearSearch,
+                ),
+            ],
+          ),
+        ),
+        const SizedBox(height: 10),
+
+        // Search Results
+        BlocBuilder<SearchCubit, SearchState>(
+          builder: (context, state) {
+            return state.when(
+              initial: () => Container(),
+              loading: () => const Center(child: CircularProgressIndicator()),
+              loaded: (results) => _buildSearchResults(results),
+              error: (message) => Center(
+                child: Text(
+                  message,
+                  style:
+                      const TextStyle(color: ThemeColors.orangeDroidconColor),
+                ),
+              ),
+            );
+          },
+        ),
+      ],
+    );
+  }
+
+  Widget _buildSearchResults(List<SearchResult> results) {
+    if (results.isEmpty) {
+      return const Text('No results found');
```

Comment:
Let the text go into the `.arb` file. 

Ref: 
- `lib/common/widgets/social_handle.dart:22`
- `lib/common/widgets/social_handle.dart:34`
- `lib/l10n/arb/app_en.arb:66`

Reply:
done